docs/developer-guide/connect.rst

```diff
@@ -0,0 +1,144 @@
+.. _ksql-connect:
+
+KSQL-Connect Integration
+========================
+
+`Kafka Connect`_ is an open source component of Kafka that simplifies loading and exporting data
+between Kafka and external systems. KSQL provides functionality to manage and integrate with Kafka
+Connect:
```

Comment:
First use of "Kafka Connect", use the `|kconnect-long|` token, and for subsequent uses, `|kconnect|`.